usaspending_api/references/views.py

```diff
@@ -94,3 +94,19 @@ def get_queryset(self):
         queryset = self.serializer_class.setup_eager_loading(queryset)
         filtered_queryset = self.filter_records(self.request, queryset=queryset)
         return filtered_queryset
+
+
+class GuideViewSet(FilterQuerysetMixin, DetailViewSet):
```

Comment:
The guide search function (which will need the autocomplete) is a story in the next sprint.